src/core/ext/transport/cronet/transport/cronet_status.h

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPC_CORE_EXT_TRANSPORT_CRONET_TRANSPORT_CRONET_STATUS_H
+#define GRPC_CORE_EXT_TRANSPORT_CRONET_TRANSPORT_CRONET_STATUS_H
+
+/*  HTTP/2 error codes are mapped to the following cronet net error codes */
+enum cronet_status_code {
+  CRONET_STATUS_SUCCESS = 0,
+  CRONET_STATUS_ILLEGAL_ARGUMENT = -100,
+  CRONET_STATUS_ILLEGAL_ARGUMENT_STORAGE_PATH_MUST_EXIST = -101,
+  CRONET_STATUS_ILLEGAL_ARGUMENT_INVALID_PIN = -102,
+  CRONET_STATUS_ILLEGAL_ARGUMENT_INVALID_HOSTNAME = -103,
+  CRONET_STATUS_ILLEGAL_ARGUMENT_INVALID_HTTP_METHOD = -104,
+  CRONET_STATUS_ILLEGAL_ARGUMENT_INVALID_HTTP_HEADER = -105,
+  CRONET_STATUS_ILLEGAL_STATE = -200,
```

Comment:
-106 INTERNET_DISCONNECTED is very useful

Reply:
Looks like -106 is not in the [Cronet_RESULT enums](https://chromium.googlesource.com/chromium/src/+/master/components/cronet/native/generated/cronet.idl_c.h)(From line 71 to 103).